programs/registry/src/protocol_config/state.rs

```diff
@@ -71,43 +79,79 @@ pub enum EpochState {
     Pre,
 }
 
+///  Diagram of epochs 0 and 1.
```

Comment:
imo can improve

suggestion:


* e.g. wrap around at 80 chars, leave empty line before next section etc. Example:

```
/// let slot = 10;
///
/// To get the latest registry epoch:
```

* label example as `example`
* make explicit that "post" is not an actual phase